src/translations/pages/assistant.ts

```diff
@@ -337,9 +337,9 @@ export const Assistant = {
         ),
       walkDistance: (distance: string) =>
         _(
-          `Total gangavstand: ${distance}`,
-          `Total walking distance: ${distance}`,
-          `Total gangavstand: ${distance}`,
+          `Total gangavstand: ${distance} m`,
+          `Total walking distance: ${distance} m`,
+          `Total gangavstand: ${distance} m`,
```

Comment:
We always keep the distance in meters I presume? 

Does the app show the distance in the abbreviated form like this or in full form?

Reply:
Always meters and like this ☝️